src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationHandler.cs

```diff
@@ -93,6 +93,10 @@ public virtual Task<bool> ShouldHandleRequestAsync()
                     {
                         return false;
                     }
+                    else if (errorContext.Result.Failure != null)
+                    {
+                        throw new InvalidOperationException("An error was returned from the RemoteFailure event.", errorContext.Result.Failure);
```

Comment:
Nit: Do we not use resources in Security?

Reply:
Not many, it's really inconsistent

Reply:
If loc ever becomes important, that will be fun :)